src/store/plugins/observables.js

```diff
@@ -41,5 +63,17 @@ export default store => {
   );
 
   // eslint-disable-next-line no-param-reassign
-  store.state.observables = { notifications: notifications$ };
+  store.state.observables = {
+    notifications: notifications$,
+    balance: balance$,
+    topBlockHeight: topBlockHeight$,
+    tokenBalance: balance$.pipe(
+      map(bal =>
+        store.state.current.token !== 0
+          ? store.state.tokens[store.state.current.token].balance
+          : bal,
+      ),
+    ),
+    balanceCurrency: balance$.pipe(map(bal => store.getters.currentCurrencyRate * bal)),
```

Comment:
The `balanceCurrency` won't be updated if `getters.currentCurrencyRate` changed. Would be more accurate to subscribe to `currentCurrencyRate` using `watchAsObservable`